### ISSUE TITLE
Add one Digit to Benchmark Summary

### ISSUE THREAD

**Issue by SebKrantz** [duckdblabs/db-benchmark#59]
Minor issue, but I'd like to suggest to add one digit to the summary table for better display of results, especially for 0.5Gb and 5Gb benchmarks.

**Comment by Tmonster** [comment]
Yes! I agree that we should have more significant digits for the smaller benchmarks. I hope I can add that soon
